src/components/card/card.styles.tsx

```diff
@@ -0,0 +1,91 @@
+import styled, { css } from "styled-components";
+import { ICardContentStyles } from "./types";
+import { DragHandleIcon } from "@lifesg/react-icons/drag-handle";
+import { Button } from "@lifesg/react-design-system/button";
+import { Color } from "@lifesg/react-design-system/color";
+import { Spacer } from "../common/spacer";
+
+export const CardContent = styled.div<ICardContentStyles>`
+    padding: 1rem;
+    border-radius: 0.5rem;
+    position: relative;
+    display: flex;
+    align-items: center;
+    box-shadow: 0px 2px 8px 0px rgba(104, 104, 104, 0.24);
+
+    :hover {
+        border: 1px solid ${Color.Primary};
```

Comment:
adding border on hover is causing the contents to jump on hover, you should add a transparent 1px border by default to prevent this